packages/core/src/fields/types/plainDate/index.ts

```diff
@@ -0,0 +1,214 @@
+
+    const usesNativeDateType = meta.provider === 'postgresql' || meta.provider === 'mysql';
+
+    const resolveInput = (value: null | undefined | string) => {
+      if (meta.provider === 'sqlite' || value == null) {
+        return value;
+      }
+      return dateStringToDateObjectInUTC(value);
+    };
+
+    const commonResolveFilter = mode === 'optional' ? filters.resolveCommon : <T>(x: T) => x;
+
+    return fieldType({
+      kind: 'scalar',
+      mode,
+      scalar: usesNativeDateType ? 'DateTime' : 'String',
+      index: isIndexed === true ? 'index' : isIndexed || undefined,
+      default:
+        typeof defaultValue === 'string'
+          ? {
+              kind: 'literal',
+              value: defaultValue,
+            }
+          : undefined,
+      map: config.db?.map,
+      nativeType: usesNativeDateType ? 'Date' : undefined,
+    })({
+      ...config,
+      hooks: {
+        ...config.hooks,
+        async validateInput(args) {
+          const value = args.resolvedData[meta.fieldKey];
+          if ((validation?.isRequired || resolvedIsNullable === false) && value === null) {
+            args.addValidationError(`${fieldLabel} is required`);
+          }
+
+          await config.hooks?.validateInput?.(args);
+        },
+      },
+      input: {
+        uniqueWhere:
+          isIndexed === 'unique'
+            ? {
+                arg: graphql.arg({ type: graphql.PlainDate }),
+                resolve: usesNativeDateType ? dateStringToDateObjectInUTC : undefined,
+              }
+            : undefined,
+        where: {
+          arg: graphql.arg({
+            type: mode === 'optional' ? PlainDateNullableFilter : PlainDateFilter,
+          }),
+          resolve: usesNativeDateType
+            ? value => commonResolveFilter(transformFilterDateStringsToDateObjects(value))
+            : commonResolveFilter,
+        },
+        create: {
+          arg: graphql.arg({
+            type: config.graphql?.create?.isNonNull
+              ? graphql.nonNull(graphql.PlainDate)
+              : graphql.PlainDate,
+            defaultValue: config.graphql?.create?.isNonNull ? defaultValue : undefined,
+          }),
+          resolve(val: string | null | undefined) {
+            if (val === undefined) {
+              val = defaultValue ?? null;
+            }
+            return resolveInput(val);
+          },
+        },
+        update: { arg: graphql.arg({ type: graphql.PlainDate }), resolve: resolveInput },
+        orderBy: { arg: graphql.arg({ type: orderDirectionEnum }) },
+      },
+      output: graphql.field({
+        type: config.graphql?.read?.isNonNull
+          ? graphql.nonNull(graphql.PlainDate)
+          : graphql.PlainDate,
+        resolve({ value }) {
+          if (value instanceof Date) {
+            return formatISO(
+              new Date(value.getUTCFullYear(), value.getUTCMonth(), value.getUTCDate()),
+              { representation: 'date' }
+            );
+          }
```

Comment:
This is OK up until year 10000,  at which point, all of this code will break anyway

```suggestion
            return value.toISOString().slice(0, 10)
          }
```